frontend/admin/src/js/lang/fr.json

```diff
@@ -1664,5 +1664,599 @@
+    "description": "Second paragraph for publish pool dialog"
+  },
+  "fOCbBa": {
+    "defaultMessage": "Résultats des compétences générales"
+  },
+  "fOl4Ez": {
+    "defaultMessage": "Mots-clés (français)",
+    "description": "Label displayed on the skill form keywords field in French."
+  },
+  "fPy7Mg": {
+    "defaultMessage": "Le français - Votre impact",
+    "description": "Label for the French - Your Impact textarea in the edit pool page."
+  },
+  "fTwl6k": {
+    "defaultMessage": "Titre précis (anglais)",
+    "description": "Label for a pool advertisements specific English title"
+  },
+  "fhkgW2": {
+    "defaultMessage": "Classification de la cible",
+    "description": "Label for a pool advertisements classification group and level"
+  },
+  "fivWMs": {
+    "defaultMessage": "Courriel",
+    "description": "Label for user table search dropdown (email)."
+  },
+  "fp7Nll": {
+    "defaultMessage": "ID du bassin #{id}",
+    "description": "Current pool breadcrumb text"
+  },
+  "fvJnoC": {
+    "defaultMessage": "Exigence linguistique : {languageRequirement}",
+    "description": "Pool advertisement language requirement"
+  },
+  "gLHr9Y": {
+    "defaultMessage": "La clôture automatique de ce bassin est programmés pour le :",
+    "description": "Third paragraph for publish pool dialog"
+  },
+  "gN5gy5": {
+    "defaultMessage": "Groupe et niveau de départ",
+    "description": "Label displayed on the pool form classification field."
+  },
+  "grrIC7": {
+    "defaultMessage": "Vous êtes sur le point de supprimer ce bassin.",
+    "description": "First paragraph for delete pool dialog"
+  },
+  "gv1Hwu": {
+    "defaultMessage": "Commencez l'affiche d'emploi vide",
+    "description": "Form header to create new pool"
+  },
+  "iOgr3Z": {
+    "defaultMessage": "Supprimer",
+    "description": "Heading for the delete pool dialog"
+  },
+  "iQzlmB": {
+    "defaultMessage": "Nouveau bassin",
+    "description": "New pool breadcrumb text"
+  },
+  "j3m2Ca": {
+    "defaultMessage": "Emplacement précis (précisez ci-dessous)",
+    "description": "Label displayed for 'specific location' option"
+  },
+  "j4G/wv": {
+    "defaultMessage": "Enregistrer les compétences utiles",
+    "description": "Text on a button to save the pool asset skills"
+  },
+  "jPlK9k": {
+    "defaultMessage": "Classification",
+    "description": "Label displayed on the edit pool form classification field."
+  },
+  "jdoFE6": {
+    "defaultMessage": "Nom du bassin et classification de la cible",
+    "description": "Sub title for pool name and classification"
+  },
+  "jiUwae": {
+    "defaultMessage": "Prolonger la date",
+    "description": "Text on a button to extend the expiry date the pool"
+  },
+  "jphtnM": {
+    "defaultMessage": "Travail à distance facultatif (recommandé)",
+    "description": "Label displayed for 'remote optional' option"
+  },
+  "jttjmJ": {
+    "defaultMessage": " la date de clôture",
```

Comment:
_Save closing date_ is the English version of this string, this is not a correct translation. Should be something like _Sauver la date de clôture_, right @gobyrne?

Reply:
Thanks for flagging this!  Gray suggested "sauveguarder la date de clôture"